### ISSUE TITLE
Circular imports

### ISSUE THREAD

**Issue by finetuned89** [sdispater/orator#153]
The [docs](https://orator-orm.com/docs/0.9/orm.html#one-to-one) describe that one can simply use syntax like:

```
class User(Model):

    @has_one
    def phone(self):
        return Phone
```

and 

```
class Phone(Model):

    @belongs_to
    def user(self):
        return User
```

In reality, this leads to NameError for the related class. When you decide to put:

    from models.phone import Phone

and 

    from models.user import User

at the top of these respective classes, you end up with an ImportError, because of circular imports. It seems that the only way to do it right is to use `import models`, and reference the model by their "full path" (e.g. `models.user.User` instead of just `User`). This works, but looks far less clean.

Is anyone else running in to this, and how can we solve this in a clean way? It would be nice to be able to use the reference in both directions.

I realize that this is mostly a "Python imports"-related question, but I mention it here because it is implied in the docs that this is how clean the code can look, and some guidance might be appropriate.

**Comment by pandasauce** [comment]
Have you tried this approach?

```python
class Phone(Model):

    @belongs_to
    def user(self):
        from models.user import User
        return User
```

**Comment by finetuned89** [comment]
Yes, that works. However, I decided to just define the relationship in just one of the models, which makes importing like `from models.user import User` work just fine. 

Thanks for the suggestion though :-)

**Comment by stefanthorpe** [comment]
To avoid circular imports error, you can relocate the imports to the bottom of the file. This way the current model loads, before the import and when the import of the relation model occurs the requirements are already meet.

**Comment by tonyzhangcn** [comment]
Same issue for me. Even though the solution provide here can solve the problem, but the solution makes the code 'ugly', there should be a more clean way. But thanks anyway!

**Comment by josephmancuso** [comment]
@tonyzhangcn the issue is how Python imports. There really can't be a better way than moving the imports into the method

**Comment by tonyzhangcn** [comment]
@josephmancuso thank you, the lucky thing is that the problem was solved :)
